chromadb/segment/impl/vector/local_hnsw.py

```diff
@@ -115,7 +116,7 @@ def get_vectors(
 
         results = []
         if self._index is not None:
-            vectors = cast(Sequence[Vector], self._index.get_items(labels))
+            vectors = cast(Sequence[Vector], np.array(self._index.get_items(labels))) # version 0.8 of hnswlib allows return_type="numpy"
```

Comment:
does `return_type="numpy"` introduce any extra overhead at the HNSW layer?

Reply:
I don't believe so, but I didn't want to version bump in the same PR so leaving a reminder to follow up on once we do. I can handle the bump if so desired.